src/content/1.7/development/mail.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Register a new hook
```

Comment:
```diff
- title: Register a new hook
+ title: How to send e-mails
```

Reply:
@mickaelandrieu thank you for reviewing my code. Is there anything may i have to do? I created the PR directly from github. How can i clone it locally now?